tests/Bundle/Routing/RoutesAttributesLoaderTest.php

```diff
@@ -544,4 +544,24 @@ public function it_generates_routes_from_resource_with_return_content(): void
             ],
         ], $route->getDefaults());
     }
+
+    /** @test */
+    public function it_generates_routes_from_resource_with_legacy_attribute(): void
+    {
+        self::bootKernel(['environment' => 'test_with_attributes']);
+
+        $container = self::getContainer();
+
+        $attributesLoader = $container->get('sylius.routing.loader.routes_attributes');
+
+        $routesCollection = $attributesLoader->__invoke();
+
+        $route = $routesCollection->get('show_book_with_legacy_attribute');
+        $this->assertNotNull($route);
+        $this->assertEquals('/book/{id}', $route->getPath());
+        $this->assertEquals([
+            '_controller' => 'app.controller.book:showAction',
```

Comment:
If it's applicable here
```suggestion
            '_controller' => 'app.controller.book::showAction',
```